chia/full_node/block_height_map.py

```diff
@@ -57,7 +57,7 @@ async def create(cls, blockchain_dir: Path, db: aiosqlite.Connection) -> "BlockH
         self.__height_to_hash_filename = blockchain_dir / "height-to-hash"
         self.__ses_filename = blockchain_dir / "sub-epoch-summaries"
 
-        res = await self.db.execute(
+        res = await self.db.db.execute(
```

Comment:
Maybe we add `DBWrapper.execute` which passes through to the connection to tidy up this weird-looking `.db.db` construct?

Reply:
yeah, that might be reasonable. I didn't introduce `DBWrapper` in this (or an earlier) patch though. I'm also a bit hesitant since these kinds of abstractions aren't exactly free in python, but I don't have a good intuition for that

Reply:
[`timeit`](https://docs.python.org/3/library/timeit.html) is good for these microbenchmarks. I doubt there is much difference, and it's not called very often anyway (maybe a few times a second tops).